spec/models/profile/assessment_answer_spec.rb

```diff
@@ -230,7 +224,7 @@
     it 'instantiates an expected AssessmentAnswer' do
       result = described_class.from_nomis_personal_care_need(personal_care_need, assessment_question, alert_type_description)
 
-      expect(result.as_json).to eq(expected_assessment_answer)
+      expect(result.as_json.symbolize_keys).to eq(expected_assessment_answer)
```

Comment:
This shouldn't be an issue in practice - JSON representation doesn't have the concept of symbols so won't affect any consumers of the API. There was some inconsistency in the serializer testing throughout the specs - in some cases the serializable_hash output was tested - which returns ruby objects, rather than generating the full JSON representation and then parsing it (a more thorough test). I've tried to make this consistent throughout the tests by always rendering to and parsing from JSON. The conversion of keys to symbols or strings is then just done to minimise noise in the spec changes.